server/admin/api.go

```diff
@@ -35,3 +42,76 @@ func (_ *Server) apiPing(w http.ResponseWriter, _ *http.Request) {
 func (s *Server) apiConfig(w http.ResponseWriter, _ *http.Request) {
 	writeJSON(w, s.core.ConfigMsg())
 }
+
+// apiMarketInfo is the handler for the '/market/{marketName}' API request.
+func (s *Server) apiMarketInfo(w http.ResponseWriter, r *http.Request) {
+	mkt := strings.ToLower(chi.URLParam(r, marketNameKey))
+	found, running := s.core.MarketRunning(mkt)
+	if !found {
+		http.Error(w, fmt.Sprintf("unknown market %q", mkt), http.StatusBadRequest)
+		return
+	}
+
+	writeJSON(w, &MarketStatus{
+		Name:    mkt,
+		Running: running,
```

Comment:
Planning to expand this to include the extended status data from `(*DEX).MarketStatus` rather than just `MarketRunning`.